cmd/server/cmd/server.go

```diff
@@ -99,18 +106,43 @@ func serverCmd(printf, fatalf shared.FormatFn) *cobra.Command {
 	// TODO: implement an option to specify how traces are reported (hardcoded to report to stdout right now).
 	serverCmd.PersistentFlags().BoolVarP(&sa.enableTracing, "trace", "", false, "Whether to trace rpc executions")
 
-	serverCmd.PersistentFlags().StringVarP(&sa.serviceConfigFile, "serviceConfigFile", "", "serviceConfig.yml", "Combined Service Config")
-	_ = serverCmd.MarkPersistentFlagFilename("serverConfigFile", "yaml", "yml")
+	serverCmd.PersistentFlags().StringVarP(&sa.configStoreURL, "configStoreURL", "", "",
+		"URL of the config store. May be fs:// for file system, or redis:// for redis url")
+	serverCmd.PersistentFlags().StringVarP(&sa.configIdentityAttribute, "configIdentityAttribute", "", "target.service",
+		"Attribute that is used to identify applicable scopes.")
```

Comment:
Made the flag hidden.
It's only purpose at present is to ensure that mixer code does not assume a particular attribute.